Core/src/org/sleuthkit/autopsy/ingest/IngestJobExecutor.java

```diff
@@ -773,51 +784,48 @@ private void startStreamingModeAnalysis() {
      * case database and streamed in, and the data source is now ready for
      * analysis.
      */
-    void startStreamingModeDataSourceAnalysis() {
+    void addStreamedDataSource() {
         synchronized (stageTransitionLock) {
             logInfoMessage("Starting full first stage analysis in streaming mode"); //NON-NLS
             stage = IngestJobExecutor.IngestJobStage.FILE_AND_HIGH_PRIORITY_DATA_SRC_LEVEL_ANALYSIS;
             currentDataSourceIngestPipeline = highPriorityDataSourceIngestPipeline;
 
             if (hasFileIngestModules()) {
                 /*
-                 * Do a count of the files the data source processor has added
-                 * to the case database. This number will be used to estimate
-                 * how many files remain to be analyzed as each file ingest task
-                 * is completed.
+                 * For ingest job progress reporting purposes, do a count of the
+                 * files the data source processor has added to the case
+                 * database. This number will be used to estimate how many files
+                 * remain to be analyzed as each file ingest task is completed.
+                 * The estimate will likely be an over-estimate, since some of
+                 * the files will have already been "streamed" to this job and
+                 * processed.
```

Comment:
estimatedFilesToProcess is not an estimate of how many files are left, it's how many files there are total. Every processed file increments the processedFiles counter regardless of whether we've finished the data source. So it may be that we switch to determinate with estimatedFilesToProcess = 100, but processedFiles is already 30 so we'd set the progress bar to 30%.

Reply:
Cool, I'll revise the comment.

Reply:
Fixed in next commit.